tests/src/test/scala/system/health/BasicHealthTest.scala

```diff
@@ -212,11 +218,10 @@ class BasicHealthTest
         case e: Exception => throw e
       }
 
-      retry({
-        println("Polling for activations")
-        val activations = wsk.activation.pollFor(N = 1, Some(triggerName), retries = maxRetries)
-        assert(activations.nonEmpty)
-      }, N = 3)
+      // Check if the trigger, that should have been created as reaction on the kafka-message, has been created.
+      // The trigger should have been created by the action, that has been triggered by the kafka message.
+      // If we cannot find it, the most probably the action did not run.
```

Comment:
maybe also add comments explaining why looking for the trigger's and/or rule action activation was problematic

Reply:
i don't think this is a good idea as this comment would just be cluttering the code documenting our prior experiences instead of explaining the behavior of the code or the logic behind the implementation, etc.